xt/pod.t

```diff
@@ -1,4 +1,6 @@
-use strict; use warnings;
+use strict;
```

Comment:
What is the benefit of changing this? A reviewer has to read through these diffs before being able to conclude they are semantically null.